examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        await sim.perform_activity("volcanic_event_detection", activity_func_args=[data_acquired[n],output_event_masks[n]])
</pre><p>Ok so on my linux machine (different one than on last review) it updates the orbit plot for 8 seconds, then freezes for quite a while and then updates 1 second. It looks completely broken and there is no discernible change in the visualization. I would suggest to do the following:</p><p> </p><ul><li>Let's fix https://github.com/aidotse/PASEOS/issues/43 in this PR so that we can actually see something in the visualization?</li><li>Continuously update the plot for all activities to avoid the impression that it freezes</li><li>Maybe you could play a bit with the times things take. It would be much more exciting to watch if something actually happened in the visualization. For demonstrative purposes you could also tweak the timesteps so that we can see some movement in the visualization. </li><li>Why are we using both satellites? We only run on the local actor, so I would completely remove Sentinel 2B from this notebook since there is 0 interaction. It is centralized example with only one sat after all? Only confusing otherwise why we have both. Also, I would suggest we invite somebody who knows paseos a bit less to try out this notebook, so we can see if they get it?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>